packages/presentation/lib/screen/home/widgets/nowShowing.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rating_bar/flutter_rating_bar.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:presentation/utils/images/paths.dart';
+import 'package:presentation/base/bloc_screen.dart';
+
+class NowShowing extends StatelessWidget {
```

Comment:
1) naming -//-
2) need to think if we can use one more generic class instead of two quite similar - Anticipated and NowShowing